crypto/build.rs

```diff
@@ -0,0 +1,17 @@
+extern crate gcc;
+
+use std::env;
+use std::path::Path;
+
+fn main() {
+    gcc::Config::new()
+        .compiler(&Path::new("arm-none-eabi-gcc"))
+        .flag("-mcpu=cortex-m4")
```

Comment:
Yes! so these asm files are based on a c implementation of a constant time implementation for m0. I rewrote the c code to be in rust, and also edited the asm code for m4 speed ups. So for m0 and m4 its constant time.

For the verifying constant-time question, the code I have doesn't have any branching and each command is suppose to be constant. I also ran many different inputs to check timings too. Not sure if there are other methods out there to check.